[13:15:07][stderr]Init stderr Log 
==== TRACEBACK <class 'UnicodeDecodeError'> ====
Traceback (most recent call last):
  File "C:\Program Files\Python313\Lib\threading.py", line 1043, in _bootstrap_inner
    self.run()
    ~~~~~~~~^^
  File "C:\Program Files\Python313\Lib\threading.py", line 994, in run
    self._target(*self._args, **self._kwargs)
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\zcveo\PycharmProjects\msgr_qws\msgr.py", line 662, in exit_wrp
    if 'exit' in input():
                 ~~~~~^^
  File "<frozen codecs>", line 325, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
==== END ====
